adv_op.c

```diff
@@ -35,22 +36,22 @@ void p_char(stack_t **stack, unsigned int line_number)
  */
 void p_str(stack_t **stack, unsigned int line_number)
 {
-	int ascii_value;
 	stack_t *node;
+	char *buff = malloc(sizeof(char *) * 8);
```

Comment:
Why (char *) ? Char * is size 8, makes this malloc large enough for 64 chars, but is statically sized. Maybe hold off on the malloc until you have the node count, and malloc to that many chars + 1